### ISSUE TITLE
Images max-width for mobile

### ISSUE THREAD

**Issue by jonpond** [Wilto/Dynamic-Carousel#44]
I modified the first section of your example: http://matmarquis.com/carousel/. However, I'm looking to get each of my images to fit a max-width of 100%, so it will proportionally fit the full width of a mobile device.

It appears fine in my desktop browser, but once I take a look at it on an iPhone, the individual images do not fit the full width of the screen. I'm not quite there yet, any ideas?

**Comment by Wilto** [comment]
Hey @jonpond! Are you using the code from that carousel demo page? If so, I would definitely recommend grabbing the latest version from the repo. That code is a lot further along than the version on http://matmarquis.com/carousel/ — hopefully that will address the issue you’re running into.

**Comment by jonpond** [comment]
Yes, I was using the demo page.  I'm using the latest now and it's working great!  Thanks again, nice work on this.  Any ideas for how to get it to loop back to the beginning after clicking through all of the slides?

**Comment by jonpond** [comment]
Nevermind...I realized you already answered my looping question over here: https://github.com/Wilto/Dynamic-Carousel/issues/45#issuecomment-2368067.  Looking forward to see how you do it!

**Comment by jonpond** [comment]
Okay, I got it working as a standalone.  However, when I put it on a page with jQuery mobile (http://code.jquery.com/mobile/1.0b3/jquery.mobile-1.0b3.min.js), jQuery mobile appears to break it.  Any ideas?
